docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -0,0 +1,149 @@
+ms.service: multiple
+---
+
+# The Azure CLI 2.0 alias extension
+
+The alias extension allows users to define custom commands for the Azure CLI by using existing commands. Aliases help keep your workflow concise and simple by allowing shortcuts and giving you the ability to use positional arguments. Since aliases are powered internally with the Jinja2 engine, they even offer advanced inline argument processing.
+
+> [!NOTE]
+> The Alias Extension is in public preview. The features and configuration file format may change.
+
+## Install the alias extension
+
+The minimum required Azure CLI version to use the alias extension is **2.0.28**. To check your CLI version, run `az --version`. If you need to update your installation, see [Install the Azure CLI 2.0](./install-azure-cli.md).
+
+Install the extension with the [az extension add](/cli/azure/extension#az_extension_add) command.
+
+```azurecli
+az extension add --name alias
+```
+
+Verify the installation of the extension with [az extension list](/cli/azure/extension#az_extension_list). If the alias extension was installed properly, you will see it in the command output.
+
+```azurecli
+az extension list --output table
+```
+
+```output
+ExtensionType    Name                       Version
+---------------  -------------------------  ---------
+whl              alias                      0.2.0
+```
+
+## Keep the extension up to date
+
+The alias extension is under active development and new versions are released regularly. New versions are not automatically installed whenever you update the CLI. Extension updates must be installed separately, using the [az extension update](/cli/azure/extension#az_extension_update) command.
+
+```azurecli
+az extension update --name alias
+```
+
+## Alias commands file format
+
+To create an alias command, you need a configuration file to hold its definition. This file is located at `$AZURE_USER_CONFIG/alias`, where the default value of `AZURE_USER_CONFIG` is `$HOME/.azure` on macOS and Linux and `%USERPROFILE%\.azure` on Windows. The alias configuration file is written in the INI configuration file format. The general format for alias commands is:
+
+```
+[command_name]
+command = invoked_commands
+```
+
+Don't include `az` as part of the invoked command.
+
+## Create simple alias commands
+
+One use of aliases is for shortening existing command groups or command names. The following example configuration shortens the `group` command group to `rg` and the `list` command to `ls`.
+
+```azurecli
+[rg]
+command = group
+
+[ls]
+command = list
+```
+
+These newly defined aliases can now be used anywhere that their definition would be.
+
+```azurecli
+az rg list
+az rg ls
+az vm ls
+```
+
+Aliases can also be shortcuts for complete commands. For example, here is an alias named  that lists available resource groups and their locations as an ASCII table:
+
+```
+[lsgroups]
+command = group list --query [].{Name:name, Location:location} --output table
```

Comment:
The query should be enclosed with a pair of quotes because it contains spaces.
`command = group list --query [].{Name:name,Location:location} --output table` or
`command = group list --query "[].{Name:name, Location:location}" --output table` would work.